sale_order_revision/view/sale_order.xml

```diff
@@ -23,6 +23,12 @@
                     string="New Revision of Quotation"
                     type="object"
                 />
+                <button
+                    name="action_cancel_create_revision"
+                    states="sale"
+                    string="New Revision"
```

Comment:
I would have added something to help user to know the current one will be canceled (in string or in help at least)